pages/apply.js

```diff
@@ -0,0 +1,120 @@
+import styled from 'styled-components';
```

Comment:
rename file `application.js` to `apply.js` so that the path will be nwplus.io/apply

Reply:
I'm so sorry I didn't get to this PR in time - I should have checked beforehand but I did this change in my PR yesterday (#107). This will probably propagate with the merge conflicts. Sorry again!